web/gridt/models/user.py

```diff
@@ -90,6 +93,15 @@ def hash_password(self, password):
         """
         self.password_hash = pwd_context.hash(password)
 
+    def hash_email(self, email):
```

Comment:
Instead I would call this function `def get_email_hash(self):`, this would then simply return the hash.